tests/snapshots/tests.contrib.django.test_django_snapshots.test_safe_string_encoding_4x.json

```diff
@@ -0,0 +1,363 @@
+[[
```

Comment:
Diff between snapshot generated by django 4 and django 2.2 for `test_safe_string_endocing`
/Users/munir.abdinur/go/src/github.com/DataDog/dd-trace-py/test_safe_string_encoding_django4_django22-diff.txt

Reply:
can you add this to the PR description as well as a change

Reply:
This is the other way around right? We lose `SafeTemplateUserList` for some reason in Django 4 and get `tests.contrib.django.views.view` instead?

Seems odd... maybe something changed in `.as_view()`? We should probably figure out why this changed since it could be breaking if a user has a facet on the resource of django view spans.